deepsearch/documents/core/create_report.py

```diff
@@ -63,6 +62,9 @@ def report_docs(
             os.path.join(tmpdir, "tmpzip/**/*.zip"), recursive=True
         )
         files_zip = files_zip + files_tmpzip
+        # if report generation is called after results are stored, they may appear as zip files.
+        # we remove them from out list.
+        files_zip = [item for item in files_zip if str(result_dir) not in item]
```

Comment:
Sorry, what is the logic here? A partial string match of `result_dir` in `item`, which is the full filename of a zip?